Dockerfile.aarch64

```diff
@@ -39,6 +39,7 @@ RUN apt-get update && apt-get install -y \
 	libcap-dev \
 	libltdl-dev \
 	libsystemd-dev \
+	libyaml-dev \
```

Comment:
To add to that: all other platforms are based on debian:jessie, while aarch64 is curently based on ubuntu:wily. So the list of packages that are being installed doesn't seem to be comparable anyway.